src/en/developer-layers.md

```diff
@@ -0,0 +1,153 @@
+
+ Title: Layers for Charm Authoring
+
+# Why build from layers
+
+When creating a charm, you always have the option of doing it by
+hand writing.  That is, you can create each hook, implement the side of the
```

Comment:
All charms a written "by hands".  These sentences can be combined to one sentence:  
When creating a charm, you always have the option of doing it traditional way by creating each hook, implementing the side of interface...